mitodl/open-discussions
# Simplify exception handling for emails

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #618 

#### What's this PR do?
Logs an exception whenever an error occurs sending an email

#### How should this be manually tested?
- Enter an invalid value for `MAILGUN_KEY` in your `.env` file
- Reply to some posts & comments
- Eventually, celery errors should appear for each email that the app tries to send.
